src/components/PageHeader/PageHeader.jsx

```diff
@@ -1,7 +1,33 @@
 import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Grid from '@material-ui/core/Grid';
+import Image from './covid.png';
+const useStyles = makeStyles(theme => ({
+  bg: {
+    height: '170px',
+    backgroundImage: `url(${Image})`,
+    backgroundSize: 'cover'
+  },
+  text: {
+    textAlign: 'center left',
+    color: 'white',
+    fontFamily: "'Code-Pro-LC', sans-serif",
+    fontWeight: 600,
+    fontSize: '30px',
+    padding: '20px '
+  }
+}));
 
 function PageHeader() {
-  return <div>Page header goes here</div>;
+  const classes = useStyles();
```

Comment:
```suggestion
  const styles = useStyles();
```
We've been using `styles` because `classes` is a more overloaded term.